utils/src/ooinstall/openshift_ansible.py

```diff
@@ -153,17 +156,17 @@ def write_inventory_vars(base_inventory, multiple_masters, lb):
 def write_proxy_settings(base_inventory):
     try:
         base_inventory.write("openshift_http_proxy={}\n".format(
-                                                            CFG.settings['openshift_http_proxy']))
+            CFG.settings['openshift_http_proxy']))
```

Comment:
I just realized this. I think this is two different things we're discussing that's causing the perceived inconsistency.

The other indentation changes you're thinking of are for functions with **multiple** parameters provided, or a **single** parameter *spanning multiple lines*. See the `collect_storage_host` function change in cli_installer.py, ~line 356:

```python
    # ...
    hostname_or_ip = click.prompt('Enter hostname or IP address',
                                  value_proc=validate_prompt_hostname,
                                  # ... more params)

```

In that case we **do** align the parameters up with the opening paren's.

Whereas in the line you pointed out in your original comment the `write` method for the `base_inventory` variable is only given 1 parameter, a formatted string:

```python
    base_inventory.write("openshift_http_proxy={}\n".format(CFG.settings['openshift_http_proxy']))
```

If we aligned the line continuation with the opening `write` method paren we would be visually hinting to the code reader that what they're seeing is another parameter.

Instead, we do a line-continuation style indent for the arguments to the `format` function, indicating this is not another parameter, just a continuation of the preceding line.